### PR TITLE
CLDR-15921 BRS v42: regen likely subtags

### PR DESCRIPTION
**Pull request by srl295**
CLDR-15921

- [ ] This PR completes the ticket.

### CODE REVIEW

File: common/supplemental/likelySubtags.xml [hunk 1]
```diff
@@ -3550,6 +3550,24 @@ not be patched by hand, as any changes made in that fashion may be lost.
 		<!--{ ?; Lanna; ? } => { Northern Thai; Lanna; Thailand }-->
 		<likelySubtag from="und_Laoo" to="lo_Laoo_LA"/>
 		<!--{ ?; Lao; ? } => { Lao; Lao; Laos }-->
+		<likelySubtag from="und_Laoo_AU" to="hnj_Laoo_AU"/>
```
Comment (macchiati) [inline common/supplemental/likelySubtags.xml:3553]: I think this is a bad result. Just from quick checks, it appears that there are far more Lao speakers in these countries than there are Hmong Njua speakers. It is just that we haven't had anyone submit data for those. I suggest that we remove these for now, and I'll file a ticket to work on a longer term solution.

File: common/supplemental/likelySubtags.xml [hunk 2]
```diff
@@ -2139,7 +2139,7 @@ not be patched by hand, as any changes made in that fashion may be lost.
 		<likelySubtag from="saf" to="saf_Latn_GH"/>
 		<!--{ Safaliba; ?; ? } => { Safaliba; Latin; Ghana }-->
 		<likelySubtag from="sah" to="sah_Cyrl_RU"/>
-		<!--{ Sakha; ?; ? } => { Sakha; Cyrillic; Russia }-->
+		<!--{ Yakut; ?; ? } => { Yakut; Cyrillic; Russia }-->
```
Comment (btangmu) [inline common/supplemental/likelySubtags.xml:2142]: At first I thought there was a problem with alphabetical order here; now I see the ordering is by code, and "Yakut" has the code "saq"; "Yakut, also known as Yakutian, Sakha, Saqa or Saxa" per https://en.wikipedia.org/wiki/Yakut_language so OK!

**Review by btangmu — DISMISSED** [review]
alphabetize Yakut?

File: common/supplemental/likelySubtags.xml (hunk 1 above)
Comment (macchiati) [reply common/supplemental/likelySubtags.xml:3553]: I opened a ticket for possible approach for the longer term.

**Review by macchiati — CHANGES_REQUESTED** [review]

File: common/supplemental/likelySubtags.xml (hunk 1 above)
Comment (srl295) [reply common/supplemental/likelySubtags.xml:3553]: makes sense.

There are still some manual reverts here already. I can file a ticket with those to make sure they are tracked.

Comment (srl295) [reply common/supplemental/likelySubtags.xml]: removed, ptal

**Review by macchiati — APPROVED** [review]

**Pull request merged**